pallets/lbp/Cargo.toml

```diff
@@ -17,6 +17,7 @@ substrate-wasm-builder = { git = "https://github.com/paritytech/substrate", bran
 [dependencies]
 codec = { default-features = false, features = ["derive"], package = "parity-scale-codec", version = "2.0.0" }
 primitive-types = { default-features = false, version = "0.8.0" }
+common-runtime = {path = '../../runtime/common', default-features = false}
```

Comment:
pallets shouldn't be dependent on the runtime, even the common one, lets use primitives for this

Reply:
Done. I have also moved `constants.rs` from `common-runtime` to `primitives-crate`.